src/styles.scss

```diff
@@ -35,3 +35,9 @@ body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
  }
 }
 
+#dashboard {
```

Comment:
@RoopashreeLakshmikeshava  Use dashboard specific styles in dashboard component scss file and also avoid using important keyword until u can not apply styles with by increasing css specificity...

Reply:
@devendra-tarento , angular material related style could not be applied when I tried adding it at component level. Hence I gave an ID to dashboard component and added the style in global styles file.